src/js/controllers/chatController.js

```diff
@@ -1,22 +1,25 @@
 'use strict';
```

Comment:
I agree, it's mostly the real "main" controller. My plan at some point is to create a controller to manage the different headers (because we might have more than 2 at some point) and to move the chat input to it's own controller that would also take care of upload and different input when we do that feature. 

I'd say next time we need more views, we should break up this controller in smaller pieces.